### PR TITLE
feat(islamic-website): community hub page markup

### PR DESCRIPTION
**Pull request by hadzhehsen**

### CODE REVIEW

File: libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+              return (
+                <IconBackground key={url} url={url}>
+                  {icon}
+                </IconBackground>
+              );
+            })}
+          </div>
+        </div>
+        <div className="absolute right-[-75px] top-[-435px] z-[-1] translate-x-1/2 translate-y-1/2 md:right-[110px] md:top-[-535px] lg:right-[175px] lg:top-[-580px] xl:right-[145px] min-[1440px]:right-[275px] min-[1440px]:top-[-585px]">
+          <BgImage />
```
Comment (olegshilov) [inline libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx]: Use next/image component as on most pages

File: libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+            </div>
+          </div>
+
+          <div className="mt-[32px] md:mt-[44px] lg:mt-[60px]">
+            <Text isMono>Join our channels here:</Text>
+          </div>
+          <div className="mt-[16px] flex flex-wrap gap-[16px] md:mt-[20px] lg:mt-[24px]">
+            {socialLinks.map(({ icon, url }) => {
+              return (
+                <IconBackground key={url} url={url}>
```
Comment (olegshilov) [inline libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx]: background with links?

File: libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+
+function IconBackground({ children, url }: PropsWithChildren<{ url: string }>) {
+  return (
+    <div className="cursor-pointer rounded-[12px] bg-[#2F2F2F] p-[16px] transition-colors duration-300 hover:bg-[#585858]">
+      <Link href={url} target="_blank" rel="noopener noreferrer">
+        {children}
+      </Link>
+    </div>
+  );
+}
```
Comment (olegshilov) [inline libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx]:
```suggestion
function SocialIconLink({ children, url }: PropsWithChildren<{ url: string }>) {
  return (
    <Link href={url} target="_blank" rel="noopener noreferrer" className="cursor-pointer rounded-[12px] bg-[#2F2F2F] p-[16px] transition-colors duration-300 hover:bg-[#585858]">
      {children}
    </Link>
  );
}
```

File: libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+  },
+];
+
+export function CommunityHubPage() {
+  return (
+    <Container>
+      <div className="relative  pb-[60px] pt-[32px] md:pb-[120px] md:pt-[52px] lg:pb-[180px] lg:pt-[68px]">
+        <div className="flex flex-col">
+          <div className="max-w-[340px] md:max-w-[400px] lg:max-w-[480px] xl:max-w-[740px] min-[1440px]:max-w-[800px]">
+            <div className="text-[46px] font-[600] leading-[52px] md:text-[60px] md:leading-none lg:text-[80px]">
```
Comment (olegshilov) [inline libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx]: this is heading

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx (hunk 4 above)
Comment (hadzhehsen) [reply libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx]: Done

File: libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx (hunk 3 above)
Comment (hadzhehsen) [reply libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx]: Done

File: libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx (hunk 2 above)
Comment (hadzhehsen) [reply libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx]: Fixed

File: libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx (hunk 1 above)
Comment (hadzhehsen) [reply libs/islamic-website/community-hub-page/src/lib/community-hub-page/community-hub-page.tsx]: Done

**Review by olegshilov — APPROVED** [review]

**Pull request merged**